content/en/docs/topics/v2_v3_migration.md

```diff
@@ -25,7 +25,7 @@ during migration:
      namespace anymore
 2. Chart repository updated:
    - `helm search` now supports both local repository searches and making search
-     queries against Helm Hub
+     queries against Artifact Hub
```

Comment:
"against the Artifact Hub"?

Reply:
The question of whether to use the definite article or not is one that features prominently in "CNCF" vs "The CNCF" as well. If we're going to say "Helm Hub" and "DockerHub" without a definite article, then we probably don't need "The Artifact Hub"?

Reply:
Good point. No need for definite article then.